data/blog/electron-5-0.md

```diff
@@ -0,0 +1,110 @@
+* `app.getFileIcon`
+* `contentTracing.getCategories`
+* `contentTracing.startRecording`
+* `contentTracing.stopRecording`
+* `debugger.sendCommand`
+* Cookies API
+* `shell.openExternal`
+* `webContents.loadFile`
+* `webContents.loadURL`
+* `webContents.zoomLevel`
+* `webContents.zoomFactor`
+* `win.capturePage`
+
+### System colors access for macOS
+
+These functions were changed or added to `systemPreferences` to access macOS systems colors:
+* `systemPreferences.getAccentColor`
+* `systemPreferences.getColor`
+* `systemPreferences.getSystemColor`
+
+### Process memory information
+
+The function `process.getProcessMemoryInfo` has been added to get memory usage statistics about the current process.
+
+### Additional filtering for remote APIs
+
+To improve security in the `remote` API, new remote events have been added so that `remote.getBuiltin`, `remote.getCurrentWindow`, `remote.getCurrentWebContents` and `<webview>.getWebContents` can be [filtered](https://github.com/electron/electron/blob/master/docs/tutorial/security.md#13-disable-or-limit-creation-of-new-windows).
+
+### Multiple BrowserViews on BrowserWindow
+
+BrowserWindow now supports managing multiple BrowserViews within the same BrowserWindow.
+
+## Breaking Changes
+
+### Defaults for packaged apps
+
+Packaged apps will now behave the same as the default app: a default application menu will be created unless the app has one and the `window-all-closed` event will be automatically handled unless the app handles the event.
+
+### Mixed sandbox
+
+Mixed sandbox mode is now enabled by default. Renderers launched with `sandbox: true` will now be actually sandboxed, where previously they would only be sandboxed if mixed-sandbox mode was also enabled.
+
+### Security improvements
+The default values of `nodeIntegration` and `webviewTag` are now false to improve security.
+
+### Spellchecker now asynchronous
+
+The SpellCheck API has been changed to provide [asynchronous results](https://github.com/electron/electron/blob/5-0-x/docs/api/web-frame.md#webframesetspellcheckproviderlanguage-provider).
+
+## Deprecations
+
+The following APIs are newly deprecated in Electron 5.0.0 and planned for removal in 6.0.0:
+
+### Mksnapshot binaries for arm and arm64
+Native binaries of mksnapshot for arm and arm64 are deprecated and will be removed in 6.0.0. Snapshots can be created for arm and arm64 using the x64 binaries.
+
+### ServiceWorker APIs on WebContents
+Deprecated ServiceWorker APIs on WebContents in preparation for their removal.
+* `webContents.hasServiceWorker`
+* `webContents.unregisterServiceWorker`
+
+### Automatic modules with sandboxed webContents
+In order to improve security, the following modules are being deprecated for use directly via `require` and will instead need to be included via `remote.require` in a sandboxed webcontents:
```

Comment:
this change is more about making the use of the `remote` module explicit due to security and performance implications.